go/proxyd/backend.go

```diff
@@ -193,16 +215,51 @@ func (b *Backend) doForward(rpcReq *RPCReq) ([]byte, error) {
 		backendErrorsCtr.WithLabelValues(b.Name, rpcReq.Method).Inc()
 		return nil, wrapErr(err, "error reading response body")
 	}
+	var responseBody RPCResponse
+	err = json.Unmarshal(resB, &responseBody)
+	if err != nil {
+		return nil, wrapErr(err, "error unmarshaling response from forward")
+	}
+	go processReponse(responseBody)
 
 	return resB, nil
 }
+func processReponse(response RPCResponse) {
```

Comment:
`*RPCResponse` to avoid copying the struct.

Reply:
I did a copy on purpose because of the go routine. Does that matter, or nah?

Reply:
Nah. You can safely pass around pointers without race conditions as long as you don't modify the data the pointer points to without a mutex.